docs/oauth.rst

```diff
@@ -160,3 +160,49 @@ Here is a sample code for your customized authentication flow
 
 .. _`GetAuthUrl()`: /PyDrive2/pydrive2/#pydrive2.auth.GoogleAuth.GetAuthUrl
 .. _`Auth(code)`: /PyDrive2/pydrive2/#pydrive2.auth.GoogleAuth.Auth
+
+
+Authentication with a service account
+--------------------------------------
+
+A *service account* is a special type of Google account intended to represent a non-human user that needs to authenticate and be authorized to access data in Google APIs.
```

Comment:
I think we need a link here to the docs https://developers.google.com/workspace/guides/create-credentials#service-account